variables.tf

```diff
@@ -62,3 +63,18 @@ variable "tags" {
   type        = "map"
   default     = {}
 }
+
+variable "s3_config" {
```

Comment:
We've opted to just reuse the description from the original docs in other modules; to make it easier for users to identify which variable in the module maps to which variable in resources. Also saves us the effort of having to write descriptions 😄 

Can just copy the most important parts from here (for the new variables introduced here):
https://www.terraform.io/docs/providers/aws/r/lambda_function.html